package.json

```diff
@@ -72,13 +80,14 @@
     "test": "wdio ./test/integration/wdio.conf.js",
     "test:integration": "wdio ./test/integration/wdio.conf.js --inspect",
     "test:integration:headless": "wdio ./test/integration/wdio-headless.conf.js",
-    "test:unit": "jest"
+    "test:unit": "jest --env=jest-environment-jsdom-sixteen"
```

Comment:
This is because we are adding the [Uprtcl](http://www.uprtcl.io/) libraries (through `@dorgtech/daosmind`), which uses custom elements created by [`LitElement`](https://lit-element.polymer-project.org/guide/start#import-your-own-litelement-component). Since we are running this tests in a node environment, the tests wont pass, because the `customElement.define()` method is defined inside the window object from browsers ([`Custom Element API`](https://developer.mozilla.org/en-US/docs/Web/API/Window/customElements))

With this library we do a "stub" of the window object, so the Custom Element API can be accessed on the tests.